test/test_autograd.py

```diff
@@ -5654,27 +5660,45 @@ def reducer(x):
         self.assertIsNotNone(res[0].grad_fn)
         self.assertIsNotNone(res[1].grad_fn)
 
-        gradcheck(lambda inp, v: autogradF.jvp(reducer, inp, v, create_graph=True), (inputs, v))
-        gradgradcheck(lambda inp, v: autogradF.jvp(reducer, inp, v, create_graph=True), (inputs, v))
+        res = autogradF.jvp(reducer, inputs, v, create_graph=True, fw_mode=False)
+        self._assert_same_struct(res[1], res[0])
+        self.assertIsNotNone(res[0].grad_fn)
+        self.assertIsNotNone(res[1].grad_fn)
```

Comment:
I'm confused -- isn't `fw_mode=False` the default right now?

Also nit:
```
for fw_mode in [True, False]:
        res = autogradF.jvp(reducer, inputs, v, create_graph=True, fw_mode=False)       
        self._assert_same_struct(res[1], res[0])
        self.assertIsNotNone(res[0].grad_fn)
        self.assertIsNotNone(res[1].grad_fn)
```

Reply:
It is, this is left from an older version. My bad!